Assets/MRTK/Core/Inspectors/Profiles/MixedRealityPointerProfileInspector.cs

```diff
@@ -111,10 +112,15 @@ public override void OnInspectorGUI()
 
                     var gazeProvider = CameraCache.Main.GetComponent<IMixedRealityGazeProvider>();
                     CreateCachedEditor((Object)gazeProvider, null, ref gazeProviderEditor);
-
                     showGazeProviderProperties = EditorGUILayout.Foldout(showGazeProviderProperties, "Gaze Provider Settings", true, boldFoldout);
                     if (showGazeProviderProperties && !gazeProviderEditor.IsNull())
                     {
+                        // Provide a convenient way to toggle the gaze provider as enabled/disabled via editor
+                        gazeProviderEnabled = gazeProvider.Enabled;
+                        gazeProviderEnabled = EditorGUILayout.Toggle("Enable Gaze Provider", gazeProviderEnabled);
+                        gazeProvider.Enabled = gazeProviderEnabled;
```

Comment:
What is the purpose of the local cache? Can we not just do `gazeProvider.Enabled = EditorGUILayout.Toggle("Enable Gaze Provider", gazeProvider.Enabled);`?

Reply:
Done